_data/guides.yml

```diff
@@ -138,6 +138,8 @@ toc:
     path: /docs/user-guide/resizing-a-replication-controller/
   - title: Service Operations
     path: /docs/user-guide/services/operations/
+  - title: Configuring Multiple Schedulers
+    path: /docs/admin/multiple-schedulers
```

Comment:
OK, thanks John! @madhusudancs can you pleae add the trailing slash, and then we can merge this PR.